lnrpc/chainrpc/chainkit.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package chainrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/chainrpc";
+
+// ChainKit is a service that can be used to get information from the
+// chain backend.
+service ChainKit {
+    // GetBlock returns a block given the corresponding block hash.
+    rpc GetBlock (GetBlockRequest) returns (GetBlockResponse);
```

Comment:
We should probably also add the `GetBlockHash` method which might be handy in some situations.